<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtool</class>
 <widget class="QMainWindow" name="vtool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>730</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>730</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>HollongApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="devicegroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Devices</string>
    </property>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>581</width>
       <height>311</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="tracesgroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Traces</string>
    </property>
    <widget class="QTabWidget" name="tracestabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>581</width>
       <height>281</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="settinggroupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>561</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QTabWidget" name="settingstabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>624</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>541</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
